sql/postgresql/init-policy-database.sql

```diff
@@ -18,7 +19,7 @@ DELETE FROM public.databasechangeloglock;
 INSERT INTO public.databasechangeloglock (ID, LOCKED) VALUES (1, FALSE);
 
 -- Lock Database
-UPDATE public.databasechangeloglock SET LOCKED = TRUE, LOCKEDBY = '127.0.0.1 (127.0.0.1)', LOCKGRANTED = '2021-09-15 17:12:02.151' WHERE ID = 1 AND LOCKED = FALSE;
+UPDATE public.databasechangeloglock SET LOCKED = TRUE, LOCKEDBY = '10.2.2.103 (10.2.2.103)', LOCKGRANTED = '2022-03-31 16:04:03.489' WHERE ID = 1 AND LOCKED = FALSE;
```

Comment:
If you decide to change the IP address in the update script, you could change it here too for consistency